core/templates/dev/head/components/concept-card/concept-card.directive.ts

```diff
@@ -35,23 +34,19 @@ angular.module('oppia').directive('conceptCard', [
       controllerAs: '$ctrl',
       controller: [
         '$scope', '$filter', '$rootScope',
-        'ConceptCardBackendApiService', 'ConceptCardObjectFactory',
+        'ConceptCardBackendApiService',
```

Comment:
Add this also to previous line, generally we follow the same formatting here and in the function parameters.